### ISSUE TITLE
Mac becomes unresponsive when running tests containing the puppetlabs/apache module

### ISSUE THREAD

**Issue by Tamerz** [puppetlabs/pdk#810]
**Describe the bug**
On MacOS (in this case 10.14.6 but I have seen this in earlier versions as well), any time you run unit tests against a module that is using the "puppetlabs/apache" module, the entire computer becomes unresponsive. Killing the tests does not resolve it, I end up having to reboot the machine.

**To Reproduce**
Create a new module, add "puppetlabs/apache" to the fixtures and then have the below in a manifest. Run 'pdk test unit':

```
class { 'apache': }
```
**Expected behavior**
Tests run as normal.

**Additional context**
- PDK installed via homebrew
- PDK version: 1.14.1
- MacOS 10.14.6

**Comment by rodjek** [comment]
Hi @Tamerz, I'm having trouble reproducing this on my end. Would you be able to tar up your test module that reproduces this behaviour and attach it to this issue?

**Comment by Tamerz** [comment]
[notworkingforme.tar.gz](https://github.com/puppetlabs/pdk/files/3890258/notworkingforme.tar.gz)
I know this is a strange one, I attributed it to my particular Mac, but now I'm seeing it on a second one. Both are work Mac's so have similar software installed, so I'm not ruling that out as well.

As a note, this time I let the computer sit for a while after killing the tests. It did eventually become responsive again without a reboot, but took some time. I also kept Activity Monitor open and didn't notice any excessive CPU or memory usage. But until then, it's all spinning beach balls and unresponsive windows.
